### PR TITLE
Add a file for Orlando, FL

### PR DESCRIPTION
**Pull request by githubteacher**
The purpose of this PR is to add a hometown file for Orlando, FL ☀️ 

Some ❓ that I want you all to help me answer:
1. If you've been to Orlando, what did you do? What did you think of it?
1. If you have kids, what kind of things do they like to do at night? 
1. What kind of food do you like?

### CODE REVIEW

**Comment by dave-burke** [comment]
Haven't been to Orlando since I was a kid. When I was a kid I slept at night.

**Comment by NatashaSteblay** [comment]
Went to Universal Studios park
Seafood

File: githubteacher-orlando.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Orlando, FL
+
+### Restaurants
+
+### Places the locals like
+
+### Places tourists like
```
Comment (dave-burke) [inline githubteacher-orlando.md:12]: Disney World, naturally. Although I think that's not technically Orlando.

**Comment by githubteacher** [comment]
This Pull Request resolves #6

File: githubteacher-orlando.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Orlando, FL
+
+### Restaurants
+- The Pharmacy is a great hidden :gem:
+- Bikurri Sushi has some great :fried_shrimp:
```
Comment (AdamPaolo) [inline githubteacher-orlando.md:5]: I suggest :sushi:

Comment (dave-burke) [inline githubteacher-orlando.md:5]: :sushi: is always a good option!

File: githubteacher-orlando.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+### Places the locals like
+- Beaches
```
Comment (dave-burke) [inline githubteacher-orlando.md]: Any particularly good beaches?

**Review by dave-burke — CHANGES_REQUESTED** [review]

**Comment by githubteacher** [comment]
@dave-burke I added specific beaches in 3773949. Does it look good to go now ?

**Review by dave-burke — APPROVED** [review]

**Pull request merged**